x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/downsample/TransportRollupIndexerAction.java

```diff
@@ -186,5 +196,42 @@ protected void finishHim() {
                 listener.onFailure(e);
             }
         }
+
```

Comment:
After some more thinking, I'm not comfortable this particular change (cancelling other shard level downsamples incase a shard level downsample fails). I think in the current state it is fragile. I don't have a good suggestion how to make this logic more robust.

Reply:
okay, I removed these code.